nuxeo-features/nuxeo-automation/nuxeo-automation-server/src/main/java/org/nuxeo/ecm/automation/server/jaxrs/batch/BatchManager.java

```diff
@@ -47,19 +47,43 @@
      * Adds an inputStream as a blob to a batch. Will create a new {@link Batch} if needed.
      * <p>
      * Streams are persisted as temporary files.
+     *
+     * @deprecated since 10.1, use {@link #addBlob(String, String, Blob, String, String)} instead
      */
+    @Deprecated
     void addStream(String batchId, String index, InputStream is, String name, String mime) throws IOException;
 
+    /**
+     * Adds an inputStream as a blob to a batch. Will create a new {@link Batch} if needed.
```

Comment:
"Adds a blob to a batch [...]" as there is no more `InputStream` in the new APIs?